test/e2e/suite_test.go

```diff
@@ -247,3 +249,9 @@ func handleFramworkFlags() {
 	flag.Parse()
 	framework.AfterReadingAllFlags(&framework.TestContext)
 }
+
+func skipIfTestingInWindowsCluster() {
```

Comment:
remove this func since blob csi driver is not supported on Windows

Reply:
done